### PR TITLE
V1.0.0 major update

### PR DESCRIPTION
**Pull request by jackbrewer**
Upgraded to work with the newest versions of [SVG Sprite](https://github.com/jkphl/svg-sprite), and [SVG to PNG](https://github.com/filamentgroup/svg-to-png).

This meant rewriting the contents of `pliers-svg-sprite.js` completely.

Removed functionality where icons with `--hover` as part of their filename were treated differently to produce modified Stylus. This is better being done by developers within a project, on a per-project basis rather than being dictated by this module.

Also added a lot of missing niceties, such as linting, tests, minor documentation and coverage reports

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+language: node_js
+node_js:
+  - 0.10
```
Comment (serby) [inline .travis.yml:4]: Add node 4

File: package.json [hunk 2]
```diff
@@ -1,25 +1,41 @@
 {
   "name": "pliers-svg-sprite",
   "version": "0.1.0",
-  "description": "",
+  "description": "Pliers wrapper for svg-sprite",
   "author": "Jack Brewer",
-  "main": "pliers-svg-sprite.js",
+  "license": "BSD",
```
Comment (serby) [inline package.json]: License should be a valid SPDX license expression. 

I would suggest switching to `ISC`

http://spdx.org/licenses/

**Comment by serby** [comment]
Looks good and works in Node.js 4.1.1

If you fancied it you could take this opportunity upgrade to the latest QA tools:

https://github.com/domharrington/generator-npm-module/blob/master/app/templates/_package.json

File: .travis.yml (hunk 1 above)
Comment (jackbrewer) [inline .travis.yml:4]: Good shout. Now testing (and passing) in 0.10, 0.12 and 4.0: https://travis-ci.org/pliersjs/pliers-svg-sprite/builds/82522140

**Comment by jackbrewer** [comment]
Nice repo! Will look at upgrading QA tools to match

**Comment by jackbrewer** [comment]
@serby Changes made to match the npm generator linting, license etc.

Had to add a rewired test to boost the coverage to 100% to pass the preset thresholds.

**Comment by serby** [comment]
Thanks. You can tweak those coverage figures if they are too high, but it
is nice to have 100%

On Monday, 28 September 2015, Jack Brewer notifications@github.com wrote:

> @serby https://github.com/serby Changes made to match the npm generator
> linting, license etc.
> 
> Had to add a rewired test to boost the coverage to 100% to pass the preset
> thresholds.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/pliersjs/pliers-svg-sprite/pull/2#issuecomment-143775450
> .

## 

paul.serby@clock.co.uk
Chief Technology Officer

_Mobile: +44 7881 550999__Skype: paul.serby_

## _Twitter: serby_
- http://www.clock.co.uk/*

The Old School HouseBridge Road, Hunton Bridge,Kings Langley, WD4 8RQ
Switchboard: +44 1923 261166www.clock.co.uk
- http://twitter.com/clock    https://github.com/clocklimited
  http://www.linkedin.com/company/clock-limited
  http://www.clock.co.uk/blog*

_www.clock.co.uk/email-disclaimer http://www.clock.co.uk/email-disclaimer_
_clock.® is a Registered Trademark of Clock Limited in the European
Community and other territories_

**Pull request merged**